setup.py

```diff
@@ -3,11 +3,8 @@
 
 _version_file = 'hooked/version.py'
 
-if sys.version_info.major < 3:
-    execfile(_version_file)
-else:
-    with open(_version_file) as vf:
-        exec(vf.read())
+with open(_version_file) as vf:
+    exec(vf.read())
```

Comment:
But then again, all we're trying to do is bring `__version__` into the local scope, so it might not matter.